Install and init errors

On install (npm install -g laravel-echo-server) i have error : 

npm WARN notarget SKIPPING OPTIONAL DEPENDENCY: No compatible version found: uws@0.12.0

On init command (laravel-echo-server init) i have error :

Error: Cannot find module '/usr/local/lib/node_modules/laravel-echo-server/node_modules/sqlite3/lib/binding/node-v51-linux-x64/node_sqlite3.node'

What i do wrong?

--- a/package.json
+++ b/package.json
@@ -33,7 +33,7 @@
     "yargs": "^5.0.0"
   },
   "optionalDependencies": {
-    "uws": "0.12.0"
+    "uws": "0.14.1"
   },
   "devDependencies": {
     "@types/node": "^6.0.60",

